packages/angular/src/selector/selector.component.html

```diff
@@ -12,16 +12,31 @@
  <!-- !model.completed || model.saving -->
  <mat-progress-bar mode="indeterminate" *ngIf="!styled"></mat-progress-bar>
 
+<!-- This shows up if the list is empty -->
+ <ng-template #emptyList>
+     <div class="selected-list-header border-default border-default-bottom">
+         <h3 [textContent]="'Selected ' + (type || 'Items')"></h3>
+     </div>
+     <div class="selected-message-box">
+         Nothing has been selected.
+     </div>
+ </ng-template>
+
+
  <!-- [ngStyle]="{'display': styled ? 'inherit' : 'none'}" -->
  <div class="selected-list" *ngIf="styled">
-    <div class="selected-list-header border-default border-default-bottom">
+    <!-- FIXME: I will probably have to move the ngIf above to the div below -->
+    <div class="selected-list-header border-default border-default-bottom" *ngIf="!empty; else emptyList">
```

Comment:
The `else` here allows an `ng-template` to be rendered if the conditional results in `false`.